includes/class-alma-wc-generic-handler.php

```diff
@@ -38,6 +45,16 @@ public function __construct() {
 		$this->settings = alma_wc_plugin()->settings;
 	}
 
+	/**
+	 * Has any child handler already rendered the widget in this page
+	 *
+	 * @return bool as rendered widget state.
+	 */
+	public function is_already_rendered() {
+
+		return self::$is_already_rendered;
```

Comment:
J'utilise personnellement le self pour les CONST et le $this->is_already_renderer pour les variables. C'est juste une remarque et je veux bien qu'on en discute 2 minutes pour que je comprenne cette différence d'approche.

Reply:
il ne s'agit pas d'une `const` mais d'une variable `static` ... on prend le temps d'en parler quand tu veux ;)